pkg/apis/aquasecurity/v1alpha1/compliance_types.go

```diff
@@ -170,7 +170,8 @@ type ComplianceCheck struct {
 // ToComplianceSpec map data from crd compliance spec to trivy compliance spec
 func ToComplianceSpec(cSpec Compliance) spec.ComplianceSpec {
 	specControls := make([]defsecTypes.Control, 0)
-	for _, control := range cSpec.Controls {
+	for i := range cSpec.Controls {
+		control := &cSpec.Controls[i]
```

Comment:
for my understanding, which rule is this and why is it helpful?

Reply:
Personally it feels a bit clunky to me to write this way. Can we take this out for now and maybe if it does really impact performance (we need to measure that), implement it in a separate PR?